### DISCUSSION TITLE
What's the change in fix ?

### DISCUSSION THREAD

**Discussion by firecerne** [UkrainianBanderasCat/Digital-Logic-Sim2#59] [Q&A] [answered]
What changes were made in the [Fix](https://github.com/UkrainianBanderasCat/Digital-Logic-Sim2/commit/3435abfa09746142ad8472cd18059a1e772e7bcf) commit ? @UkrainianBanderasCat replaced all the code, I have no idea of what he changed.

**Comment by UkrainianBanderasCat** [comment]
When I merged copy-paste it didn't merge code , so I did it manually

**Comment by AlphaCodingPilot** [comment, marked as answer]
I think all of my stuff is in "fix"
